services/horizon/internal/actions/liquidity_pool.go

```diff
@@ -61,13 +61,14 @@ func (handler GetLiquidityPoolByIDHandler) GetResource(w HeaderWriter, r *http.R
 // LiquidityPoolsQuery query struct for liquidity_pools end-point
 type LiquidityPoolsQuery struct {
 	Reserves string `schema:"reserves" valid:"optional"`
+	Account  string `schema:"account" valid:"optional"`
 
 	reserves []xdr.Asset
 }
 
 // URITemplate returns a rfc6570 URI template the query struct
 func (q LiquidityPoolsQuery) URITemplate() string {
-	return "/liquidity_pools?{?reserves}"
+	return "/liquidity_pools?{?reserves,?account}"
```

Comment:
I think we don't prepend with `?` in any other template.
```suggestion
	return "/liquidity_pools?{?reserves,account}"
```